### ISSUE TITLE
Blank HelloZeroNet pannel

### ISSUE THREAD

**Issue by DaniellMesquito** [HelloZeroNet/ZeroNet#556]
After I've download the Zero ID, the pannel page is:
<img src="http://image.prntscr.com/image/0aa4400592f94276bcbea3df3e92a28d.png"/>

**Comment by HelloZeroNet** [comment]
Sorry, it's not possible to help you based on the provided informations. Please give more information: JS console messages, ZeroNet log messages, OS, Browser, etc.

**Comment by weakish** [comment]
I got the same issue.

JS console:

```
all.js:2698 Uncaught TypeError: Cannot read property 'length' of undefined
```

The line in `all.js` is:

```js
title: this.row.content.title.length > 20 ? this.row.content.title : void 0
```

So the issue is some sites do not have a title?

**Comment by DaniellMesquito** [comment]
@weakish
> I got the same issue.
> 
> JS console:
> 
> all.js:2698 Uncaught TypeError: Cannot read property 'length' of undefined
> The line in all.js is:
> 
> title: this.row.content.title.length > 20 ? this.row.content.title : void 0
> So the issue is some sites do not have a title?

All my problems the people relativize saying that the cause is my side. Happy to see that the problem isn't me! Thanks for commenting. Now I'll see the debug data and post here. My problem started after download Zero ID, I think.

**Comment by HelloZeroNet** [comment]
Thanks, based on the js error i was able to reproduce/fix it

**Comment by weakish** [comment]
On my machine, this site is the clause: `http://127.0.0.1:43110/1ARCHYFfke9k3Jbj9uVXmbeDV9XueK3hLA`.
This site is a just a place holder for selling domain and I think it is safe to delete it.

I stopped zeronet, remove `date/1ARCHYFfke9k3Jbj9uVXmbeDV9XueK3hLA` and its entry in `sites.json`.
Then I restarted zeronet, now ZeroHello comes back.

**Comment by DaniellMesquito** [comment]
@weakish, I see now the log and had the same error. Thanks for help. I never hear talk about "date" folder, but "data" is where the sites are placed. 1ARCHYFfke9k3Jbj9uVXmbeDV9XueK3hLA is my site, why?! I'm sure you seen my site in another issue and going here to talk that the cause of problem is my side, but I'm not the reason of the problem!
@HelloZeroNet, thanks for fix and reproduce it. Believe me, I don't report here wrong feedback, I'm here for help.

**Comment by DaniellMesquito** [comment]
@HelloZeroNet, whats the RV version of this fix update?

**Comment by HelloZeroNet** [comment]
@DaniellMesquito You don't need to upgrade ZeroNet, the change is only in ZeroHello site's source code which should happen automatically.

**Comment by DaniellMesquito** [comment]
@weakish 
> On my machine, this site is the clause: http://127.0.0.1:43110/1ARCHYFfke9k3Jbj9uVXmbeDV9XueK3hLA.
> This site is a just a place holder for selling domain and I think it is safe to delete it.
> 
> I stopped zeronet, remove date/1ARCHYFfke9k3Jbj9uVXmbeDV9XueK3hLA and its entry in sites.json.
> Then I restarted zeronet, now ZeroHello comes back.
This isn't only safe to delete, but safe to keep, too. This site is totally safe. Like any site in ZeroNet while using sandbox. The major utility of this site was serve as reference for fix this error.

**Comment by weakish** [comment]
@DaniellMesquito 

>  I never hear talk about "date" folder

Thanks for pointing out the typo.  Fixed now.

> This site is totally safe. Like any site in ZeroNet while using sandbox.
1. Yes the site is safe. (It is just some plain HTML, it is safe even if not sandbox.) I mean deleting this site will not affect other sites (e.g. zeroid.bit affected lots of sites).

> how can you see a debug log better than me and say have knowledge to say my site is the problem? 
> I'm sure you seen my site in another issue and going here to talk that the cause of problem is my side

I just search the "title" key in content.json of every site (via shell scripting), and found that site.
I did not seen the site in another issue, and I didn't know who own the site before.
Deleting a site is just a temporary workaround.
`this site is the clause` is not accurate. I'm in a hurry.
The better wording is `this site triggers a bug in ZeroHello`.
